climpred/metrics.py

```diff
@@ -372,6 +587,20 @@ def _brier_score(forecast, reference, **metric_kwargs):
     return brier_score(func(reference), func(forecast).mean('member'))
 
 
+_brier_score = Metric(
```

Comment:
It would be good to keep all of these in the order of the arguments from the main class.